go/sqltypes/value.go

```diff
@@ -435,6 +437,78 @@ func (v Value) String() string {
 	return fmt.Sprintf("%v(%s)", Type(v.typ), v.val)
 }
 
+// ToTime returns the value as a time.Time in UTC.
+// NULL values are returned as zero time.
+func (v Value) ToTime() (time.Time, error) {
+	return v.ToTimeInLocation(time.UTC)
+}
```

Comment:
If nothing uses this, I think we should remove this one and be explicit everywhere about the timezone needed. 

Always assuming UTC is a bit of a foot gun, since how MySQL itself works is that the local timezone is always used. 

People should really use UTC also as the local zone but there's no guarantee that it is the same.

Reply:
Fair enough, I'll make the change